feat: Add App Attestation Service (#121)
italia/eidas-it-wallet-docs: docs/en/wallet-instance-attestation.rst

   :target: https://www.plantuml.com/plantuml/ZP91RzH038NlyojCJwr4n7qFgrOSAf2G409wwSL9h60ryGmUpqRRNuyt6qBJe5MlzlFtx3TpcmtLoj27Tqcn6n2CuZEO5WfOB4ePQj8GagkuuOHYSFKZaru1PYZh-WFsFHby4eTAGvDavFzglceyS3jZndgjkKi9q8mSOnm5tEx0Cy_h8HIezaxUkHKROy_F1A_C7oKgAFqkJlcGb38vkL5gIKuJEOnSxSTw1_S-z6ef6CYmHSCmrfMhtEZBN84cYY4BI_U21dPCbD_34nqdJrOQlECLaZP55flzdFJJrtKIRKnDIpQN_RtjdeJKXHCr8MkUcsYsWs_dqq2Y7nky1DLvRguiVX-Lq3RnmDs_V1VMvuVl0HlZmsbWh5SHuGlzzHjWDwVizZwrlNWPwqWA2mdb3DVJsZUdIwh9rML6dR8TeVb5pHCevTAROy_jXPgv4xIYjBIMv53QgNtf-kMDBuishtT1tD8wHUUNBPwNlzi-YXAsHx08iJPa0Q5nzLjlITeoz7y0

- **Message 1**: The User starts the Wallet Instance mobile app, a new Wallet Instance Attestation is automatically obtained if the previous one results expired.
- **Message 2-3**: The Wallet Instance retrieves metadata about its Wallet Provider, including the list of supported algorithms, public keys, and endpoints.
- **Message 4**: The Wallet Instance verifies the Wallet Provider's trustworthiness by resolving the provider's trust chain to the Trust Anchor.
- **Message 5**: The Wallet Instance generates a new key pair.

> - **Message 6-7**: The wallet instance requests a ``nonce`` from the App Attestation Service.

Reviewer: ```suggestion
- **Message 6-7**: The Wallet Instance requests a ``nonce`` from the App Attestation Service.
```

To obtain a Wallet Instance Attestation from the Wallet
Provider it is necessary to send a Wallet Instance Attestation
Request from the Wallet Instance containing the associated public key

> and a ``nonce`` previously requested. A ``jwti`` is used to avoid replay attacks.

Reviewer: ```suggestion
, the ``nonce`` value previously requested and a ``jti`` value.
```

Reviewer: @grausof considering we already have the nonce, is the jti required?
I mean that by just using the nonce the replay attacks can be avoided

||       ||                                                            |
+--------+-------------------------------------------------------------+
|| nonce || The nonce obtained from the                                |

> ||       || App Attestation Service.                                   |

Reviewer: Ok, can I ask you to add the App Attestation Service here
https://github.com/italia/eudi-wallet-it-docs/blob/versione-corrente/docs/en/defined-terms.rst#defined-terms

Author: Already added https://github.com/italia/eudi-wallet-it-docs/pull/121/files#diff-0b13b354d3cc69c57166e38bca0c889a5e024419acaed05995f6c1ffe5d3b6e4R40
